docs/pages/blog/mui-core-v5.md

```diff
@@ -0,0 +1,546 @@
+
+The first immediate benefit was at the **performance** level. the `<Box>` component is [x5-x10 more performant](https://codesandbox.io/s/zlh5w?file=/src/App.js) in v5, compared to v4.
+
+We would like to thanks all the community contributors that made the migration of the components and documentation possible in [#24405](https://github.com/mui-org/material-ui/issues/24405) and [#16947](https://github.com/mui-org/material-ui/issues/16947) @natac13, @vicasas, @mngu, @kodai3, @xs9627, @povilass, @duganbrett, @vinyldarkscratch, and more.
+It was a major undertaking!
+
+Going forward, developers can either keep using JSS with the legacy `@mui/styles` package [or migrate](/guides/migration-v4/#migrate-makestyles-to-emotion).
+We recommend the latter to match the core components.
+
+### The `sx` prop
+
+While the `styled` API is great to style complex components or to create highly reused components, there are cases where it's overkill.
+We started to [explore](https://medium.com/material-ui/introducing-material-ui-design-system-93e921beb8df) this **problem** three years ago with the introduction of the `<Box>` component to solve:
+
+1. **Switching context** wastes time.
+   The styled API forces to constantly jump between the usage of the styled components and where they are defined.
+   Could we move the style descriptions right where we need them?
+2. **Naming things** is hard.
+   Have you ever found yourself struggling to find a good name for a styled component?
+   Could we remove the need to create and name yet another component?
+3. **Enforcing consistency** in UIs is hard.
+   This is especially true when more than one person is building the application, as there has to be some coordination amongst members of the team regarding the choice of design tokens and how they are used, what parts of the theme structure should be used with what CSS properties, and so on.
+
+In v5, we have pushed the solution one step further with the `sx` prop.
+The prop is now available on **all** the components thanks to the migration away from JSS.
+It exposes as a superset of the CSS API, the normal CSS properties, shorthands, and media query helpers.
+
+For instance, you can add two vertical margin units with:
+
+```jsx
+// add margin: 16px 0px;
+<Slider sx={{ my: 2 }} />
+```
+
+<p class="blog-description"><a href="https://codesandbox.io/s/nostalgic-williams-zmo5r?file=/src/App.js">Codesandbox</a></p>
+
+Developers seem to already [love it](https://twitter.com/AnsonLowZF/status/1397034690771443715).
+You can find a [side-by-side comparison](/system/basics/#why-use-the-system) of `styled` vs. `sx` in the documentation to determine when you want to use the prop.
+Find where your cursor is. Some developers use it for everything, others with parsimony.
```

Comment:
> Find where your cursor is

😕

> parsimony

This is very likely the first time I see this word ;)

Reply:
Ok, I have clarified it.